src/home/homework-luks.c

```diff
@@ -1377,7 +1386,14 @@ int home_setup_luks(
                 if (!subdir)
                         return log_oom();
 
-                r = luks_validate(setup->image_fd, user_record_user_name_and_realm(h), h->partition_uuid, &found_partition_uuid, &offset, &size);
+                r = luks_validate(
+                        setup->image_fd,
+                        user_record_user_name_and_realm(h),
+                        h->partition_uuid,
+                        h->luks_sector_size == UINT64_MAX ? UINT32_MAX : user_record_luks_sector_size(h), /* if sector size is not specified, select UINT32_MAX, i.e. auto-probe */
+                        &found_partition_uuid,
+                        &offset,
+                        &size);
```

Comment:
we indent function parameters to twice the regular indentation, according to our coding style